jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1865,11 +2082,30 @@ public VarHandle unreflectVarHandle(Field field) throws IllegalAccessException {
 			 * as it is not set up for lookup objects by default.
 			 */
 			int fullAccessMode = FULL_ACCESS_MASK | MODULE | UNCONDITIONAL;
+			
+			if ((dropMode != PUBLIC)
+					&& (dropMode != MODULE)
+					&& (dropMode != PACKAGE)
+					&& (dropMode != PRIVATE)
+					&& (dropMode != PROTECTED)
+					&& (dropMode != UNCONDITIONAL)
+			) {
```

Comment:
This is a good use case for a switch statement:
```java
switch(dropMode) {
case PUBLIC:
case MODULE:
case PACKAGE:
case PRIVATE:
case PROTECTED:
case UNCONDITIONAL:
  /* dropMode is OK */
  break;
default:
	/*[MSG "K065R", "The requested lookup mode: 0x{0} is not one of the existing access modes: 0x{1}"]*/
 	throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K065R", Integer.toHexString(dropMode), Integer.toHexString(fullAccessMode))); 
}
```

Reply:
Agreed and replaced with switch statement.